src/main/webapp/2Dscript.js

```diff
@@ -505,6 +594,23 @@ function getPolyLine(linePoints, color, opacity, weight, index = 1) {
   });
   return polyLine;
 }
+/**
+ * Clear prior markers/path if they exist.
+ * @param {Array<google.maps.Polyline | google.maps.InfoWindow |
+ * google.maps.Marker>} pathArray
```

Comment:
Is it okay for the type info to break across lines like that? It looks odd with Github's syntax highlighting.

Reply:
It highlights it properly in vsCode, believe it is acceptable as there's also that 80 line word limit.